compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -1031,16 +1031,16 @@ static void addLowerToLLVMGPUPasses(OpPassManager &modulePassManager,
       .addPass(createSCFToControlFlowPass)
       .addPass(createCanonicalizerPass)
       .addPass(createCSEPass)
-      .addPass(memref::createFoldMemRefAliasOpsPass)
+      .addPass(createIREECodegenFoldMemRefAliasOpsPass)
```

Comment:
So, as an alternative ... why not move this pass before scf-to-cf, so that the `scf-to-cf` becomes basically the last thing in the pipeline